### DISCUSSION TITLE
licenses don't work (The license is not installed)

### DISCUSSION THREAD

**Discussion by sidyoriginal** [community/community#122695] [Repositories] [answered]
https://github.com/community/community/assets/156057397/363bdb47-3e4d-4f04-8c1f-614ad4a6fbf2

**Comment by davevad93** [comment, marked as answer]
Hi @sidyoriginal ,

That's a strange behavior indeed, I don't know what could be the reason, have you tried to create the repository without a license and then add it later? Try to follow these steps:

1. Go to your new repo and click on `Add File` -> `Create new file` 
![1](https://github.com/community/community/assets/94308234/9c820dcd-b32b-4584-bac1-bba055720a13)

2. Name it `LICENSE.md` and then click on `Choose a license template`
![2](https://github.com/community/community/assets/94308234/ff4c96e9-9039-425b-a945-a0ca8523759f)

3. Choose a license
![3](https://github.com/community/community/assets/94308234/8f32ee85-6b94-4624-8aa4-d6b2be0c2650)

Check if you are able to create a license for your repo this way.
